src/functions/asyncCredential/sessionService/sessionService.ts

```diff
@@ -11,7 +13,7 @@ import {
   successResponse,
 } from "../../types/errorOrValue";
 
-export class SessionService implements IRecoverAuthSession {
+export class SessionService implements ISessionService {
   readonly tableName: string;
   readonly indexName: string;
```

Comment:
Please may we tidy the naming up - you can have multiple indices

Reply:
We actually had a bit of a discussion about this, would be good to chat with you about it too when you have a sec

Reply:
Done